README.md

```diff
@@ -68,18 +86,15 @@ There are three major modules:
 - ttypes.py
 
 Please refer to the [sample code](examples/ClientExample.py) on detail usage.
+If you want to run the sample code, please install `prettytable` via pip.
 
 - Steps to create a client
   - Step1: create a connection pool
+    - The default connection number of the connection pool is **two**
+    - The default timeout of connection is **1000ms**
+    - When the number of created clients exceeds the number of connections in the connection pool, more than one client will enter the wait state and wait for the previous client to release before continuing
```

Comment:
When the created client exceeds the number of connections in the connection pool, the client that exceeds the number of connections will enter the wait state, waiting for the previous client to release before continuing。